### DISCUSSION TITLE
vue-admin-box使用交流区

### DISCUSSION THREAD

**Discussion by cmdparkour** [cmdparkour/vue-admin-box#34] [Announcements]
大家在使用过程中遇到的问题都可以在这个地方记录一下，说不定其他小伙伴也会有遇到过相同的问题，互相帮助是开源项目的成长动力。

**Comment by Ero-Cat** [comment]
踩坑 在npm run dev报错 ， 换了3次node环境。（最后降到14.19.1成功可用）貌似vite对Npm高版本不支持？

**Comment by Ero-Cat** [comment]
推荐新建个仓库做demo
